File: week-2/w2-4.js
function twoSum(nums, target){
    for(let x=0; x<nums.length;x++){
        let goal=target-nums[x];
        for(let y=x+1; y<nums.length; y++){
            if(nums[y]===goal)
            return [x,y];
        }
    }
}
    let result=twoSum([2, 11, 7, 15], 9);
    console.log(result);// show [0, 2] because nums[0]+nums[2] is 9

File: week-2/w2-4.py
def twoSum(nums, target):
    numMap = {}
    for i in range(len(nums)):
        if numMap.__contains__(target-nums[i]):
            return [numMap.get(target-nums[i]), i]
        else:
            numMap[nums[i]] = i
result=twoSum([2, 11, 7, 15], 9)
print(result) # show [0, 2] because nums[0]+nums[2] is 9